ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Embed/Nary.hs

```diff
@@ -226,7 +228,7 @@ injectInitialExtLedgerState cfg extLedgerState0 =
         State.align
           (InPairs.requiringBoth
             (hmap (WrapConsensusConfig . configConsensus) cfgs)
-            (translateChainDepState hardForkEraTranslation))
+            (InPairs.hmap (error "TODO") $ translateChainDepState hardForkEraTranslation))
```

Comment:
This is the `ChainDepState` bit I didn't handle.

Reply:
Implemented this using `extendToHorizon` as added in the latest push.